spec/dynamoid_advanced_where/node_tests/equality_spec.rb

```diff
@@ -32,11 +33,25 @@
       expect(klass.where{ native_bool == true }.all).to match_array([item1])
     end
 
-    it "matches true" do
+    it "matches false" do
       expect(klass.where{ native_bool == false }.all).to match_array([item2])
     end
   end
 
+  describe "default boolean equality" do
+    let!(:item1) { klass.create(default_bool: true) }
```

Comment:
`true_item` `false_item` `nil_item` makes specs clearer and easier to read, not a blocker for merge.

Reply:
agreed I'll update them